mongoose.c

```diff
@@ -2870,7 +2870,9 @@ static int mg_accept_conn(struct mg_connection *lc) {
   /* NOTE(lsm): on Windows, sock is always > FD_SETSIZE */
   sock_t sock = accept(lc->sock, &sa.sa, &sa_len);
   if (sock == INVALID_SOCKET) {
-    if (mg_is_error(-1)) DBG(("%p: failed to accept: %d", lc, errno));
+    if (mg_is_error(-1)) {
```

Comment:
while I personally like explicit braces, omitting them is not so brittle as you imply here.

```sh
$ gcc -DCS_NDEBUG mongoose.c
```

```c
...
  sock_t sock = accept(lc->sock, &sa.sa, &sa_len);
  if (sock == (-1)) {
    if (mg_is_error(-1)) ;
    return 0;
  }
...
```

As you can see the `;` is not part of the macro and thus creates an (empty) statement regardless of whether `DBG` expands to nothing.

Please remove this commit from this PR